app/scripts/lib/assertion.js

```diff
@@ -24,14 +24,13 @@ function (P, jwcrypto) {
     // browser has native crypto, no need to fetch entropy from the server.
     try {
       if (window.crypto && window.crypto.getRandomValues) {
-        return P(true); /*jshint ignore:line*/
+        return P(true);
       }
     } catch(e) {
       // some browsers blow up when trying to query window.crypto.
     }
 
     // wah wah, we need to get entropy from the server.
-    //jshint validthis: true
```

Comment:
@pdehaan Would you be adding that to the config-fxa? @IanVS Thank you for the heads up :)